README.md

```diff
@@ -5,7 +5,7 @@
 Add the plugin with the command below in your cordova project directory.
 
 ```
-cordova plugin add https://github.com/jellyfin/cordova-plugin-chromecast.git
+cordova plugin add https://github.com/qualabs/cordova-plugin-chromecast.git
```

Comment:
This change probably isn't supposed to be in the pull-request ;)

Reply:
Got it. It's fixed!